worker/include/RTC/SimulcastConsumer.hpp

```diff
@@ -101,6 +101,7 @@ namespace RTC
 		std::vector<RTC::RtpStreamSend*> rtpStreams;
 		std::vector<RTC::RtpStream*> producerRtpStreams; // Indexed by spatial layer.
 		bool syncRequired{ false };
+		bool lastSentRtpPacketHasMarker{ true };
```

Comment:
I would say this should be initialized to *false*

Reply:
If initialized to `true`, the first RTP packet the receiver endpoint will receive will be an empty packet. IMHO it's better to have it initialized to true so we avoid sending such a useless packet when the consumer sends its first packet.

Said that, I'm not sure if we should also unset this flag in other circumstances (when the consumer is paused/resumed, etc) or not.

Reply:
> If initialized to `true`, the first RTP packet the receiver endpoint will receive will be an empty packet. IMHO it's better to have it initialized to true so we avoid sending such a useless packet when the consumer sends its first packet.
> 
> Said that, I'm not sure if we should also unset this flag in other circumstances (when the consumer is paused/resumed, etc) or not.

You are right, that's why we inited it as true. we will do some test tomorrow see what will happen when pasuing and resuming.

Reply:
:+1: